src/components/About/About.module.css

```diff
@@ -1,3 +1,173 @@
 .title {
   text-align: center;
 }
+
+@keyframes ldio {
+  0% { 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5763907)